site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -220,20 +221,6 @@ const getDefaultBuildParameters = (
   return buildValues;
 };
 
-export const orderedTemplateParameters = (
```

Comment:
Didn't see this function get used anywhere, but also, one concern I have with the function is that even though it groups the mutable/immutable params, they're still all in one array, so you have to iterate through it to see where one starts, and the other stops

I feel like, if this does need to get added back down the line, two changes would be ideal:
1. Returning this as an object (`{ immutable, mutable }`, so that you know which one is which from a glance
2. Use some custom type predicates under the hood, so that TypeScript is able to say that the `mutable` property is always `true` inside the `mutable` array, and always `false` inside the `immutable` array